packages/11ty/content/_assets/styles/print.scss

```diff
@@ -1,4 +1,6 @@
 @use "sass:math";
+@import "colors";
+@import "variables";
```

Comment:
@anderspollack we may want to use `@use` rather than `@import`

Reply:
@mphstudios for some reason I was getting this error with `@use` -- it should be able to find `variables.scss` but `$white` is still missing somehow

```
Error: Undefined variable.
   ╷
37 │ $quire-cover-text-color: $white;
   │                          ^^^^^^
   ╵
  .11ty-vite/_assets/styles/variables.scss 37:26   @use
```